### DISCUSSION TITLE
Protein-Protein-Water-Docking

### DISCUSSION THREAD

**Discussion by Jason-Han-Hu** [chaidiscovery/chai-lab#86] [Q&A] [answered]
I performed a docking of two proteins with water using its chemical formula 'H2O' [assigned Copies 500], but no water molecules appeared in the output PDB file when viewed with PyMOL. Could there be an issue with my configuration on web server?

![image](https://github.com/user-attachments/assets/50bf2e55-8b48-49e2-a5db-22aa9154a037)

**By the way**, I also tried assigning water using its SMILES code 'O', but encountered a warning about invalid characters.
![image](https://github.com/user-attachments/assets/1dc33c80-962f-41d0-a595-c65d9477c1f9).

**Web server doesn't include water in ligand?**

**Comment by joshim5** [comment, marked as answer]
HI @Jason-Han-Hu, thanks for the question. We remove all water molecules during training and inference. Since the model is trained without the water molecules (but they are inherently present in the training data), the predictions will also assume that water is present (although the water molecules won't be shown explicitly). 

If you have an application that requires water to be explicitly shown, we'd love to hear more.

**Comment by Jason-Han-Hu** [comment]
Thanks for you response, @joshim5.

 I'm trying to identify key amino acids involved in protein-protein interactions, so I need to visualize all possible hydrogen bonds, salt bridges, or other interactions. Is there a way to explicitly display these water molecules in PyMOL?
